Add the certificates from the ca-certificates package to the path where gnutls
look for certificates


Index: vlc-debian/modules/misc/gnutls.c
===================================================================
--- vlc-debian.orig/modules/misc/gnutls.c	2009-05-12 13:55:44.000000000 +0200
+++ vlc-debian/modules/misc/gnutls.c	2009-05-12 14:36:22.000000000 +0200
@@ -695,6 +695,10 @@
         sprintf (path, "%s/ssl/certs/ca-certificates.crt", confdir);
         gnutls_Addx509File (VLC_OBJECT (p_session),
                             p_sys->x509_cred, path, false);
+    gnutls_Addx509File( VLC_OBJECT(p_session),
+                        p_sys->x509_cred,
+                        "/etc/ssl/certs/ca-certificates.crt",false );
+
     }
     p_session->pf_handshake = gnutls_HandshakeAndValidate;
     /*p_session->pf_handshake = gnutls_ContinueHandshake;*/
